### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SaperZP**
- [DEMO LINK](https://saperZP.github.io/layout_calendar/)
- [TEST REPORT LINK](https://saperZP.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by EvgenBabin1 — APPROVED** [review]
good job

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  flex-wrap: wrap;
+  gap: $day-gap;
+  max-width: $cal-max-width;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+    width: $calendar-width;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss]:
```suggestion
    height: $calendar-size;
    width: $calendar-size;
```

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    border: 1px solid black;
+    font-size: 30px;
+    cursor: pointer;
+    background-color: #eee;
+    transition: 0.5s;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss]: Always add a transition for styles that you want smoothly change

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    &:hover {
+      transform: translate(0,-20px);
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss:20]:
```suggestion
      transform: translateY(-20px);
```

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      background-color: pink;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day:nth-child(#{$i})::before {
+      content: "#{$i}";
+      color: black;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss]: you can add color for .calendar__day class

File: src/styles/utils/reset.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+}
```
Comment (AlinaMatuschak) [inline src/styles/utils/reset.scss]:
```suggestion
.container {
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100vh;
}
```

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Almost perfect 👍

File: src/styles/blocks/calendar.scss (hunk 1 above)
Comment (SaperZP) [reply src/styles/blocks/calendar.scss]: partially implemented

File: src/styles/blocks/calendar.scss (hunk 3 above)
Comment (SaperZP) [reply src/styles/blocks/calendar.scss:20]: not sure

File: src/styles/blocks/calendar.scss (hunk 2 above)
Comment (SaperZP) [reply src/styles/blocks/calendar.scss]: Good suggestion. Fully implemented

File: src/styles/blocks/calendar.scss (hunk 4 above)
Comment (SaperZP) [reply src/styles/blocks/calendar.scss]: done

File: src/styles/blocks/calendar.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      background-color: pink;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day:nth-child(#{$i})::before {
+      content: "#{$i}";
+      color: $day-color;
+    }
+  }
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss:30]: you anywhere add color for all days, so add color for calendar__day class

```suggestion
  &__day {
    display: flex;
    justify-content: center;
    align-items: center;
    height: $calendar-height;
    width: $calendar-width;
    border: 1px solid black;
    font-size: 30px;
    cursor: pointer;
    background-color: $day-backgroudColor;
    transition: transform 0.5s, background-color 0.5s;
    color: $day-color;

    &:hover {
      transform: translateY(-20px);
      background-color: pink;
    }
  }

  @for $i from 1 through 31 {
    &__day:nth-child(#{$i})::before {
      content: "#{$i}";
    }
  }
```

File: src/styles/blocks/calendar.scss (hunk 3 above)
Comment (AlinaMatuschak) [reply src/styles/blocks/calendar.scss:20]: you don’t move your element on the x-axis, so use translateY instead of translate

**Review by AlinaMatuschak — APPROVED** [review]

**Pull request closed without merging**